logging/src/posix/logging.cc

```diff
@@ -6,7 +6,8 @@ using namespace std;
 
 namespace leatherman { namespace logging {
 
-    void colorize(ostream& dst, log_level level)
+    template <typename StreamT>
+    void colorize_impl(StreamT& strm, log_level level)
```

Comment:
Why would we ever colorize something that wasn't an `ostream`? Does boost have streams that don't inherit from the normal std bases?